src/abundances.jl

```diff
@@ -1,62 +1,59 @@
 # Methods for absolute and relative abundances
 
-struct AbundanceTable{T<:Real} <: AbstractArray{T,2}
-    table::Array{T,2}
-    samples::Vector{S} where S
-    features::Vector{R} where R
-end
-
-function AbundanceTable(df::DataFrame)
-    return AbundanceTable(
-            Matrix(df[:,2:end]),
-            names(df[2:end]),
-            Vector(df[1]))
-end
-function AbundanceTable(table::Array{T,2}) where T<:Real
-    return AbundanceTable(
-             table,
-             Vector{Int64}(1:size(table,2)),
-             Vector{Int64}(1:size(table,1)))
-end
-
-@forward_func AbundanceTable.table Base.getindex, Base.setindex, Base.length, Base.size
+abundancetable(df::DataFrame) = ComMatrix(df, sitecolumns = true)
+abundancetable(table::Array{T,2}, site = ["sample_$x" for x in indices(table, 2)],
+    species = ["feature_$x" for x in indices(table, 1)]) where T<:Real =
+    ComMatrix(table', species, site)
 
+@forward_func ComMatrix.occurrences Base.length
```

Comment:
The others are defined already in SpatialEcology. Maybe length should be too? What do you use it for?

Reply:
I don't think I use length in this context typically, though I do use `size()`.

Reply:
size is defined for ComMatrix already